### DISCUSSION TITLE
Proper configuration of backups

### DISCUSSION THREAD

**Discussion by ghost** [cloudnative-pg/cloudnative-pg#4668] [Q&A]
Hello,
I have configured a pg-cluster which has access to an S3-Bucket. WALs are generated, a base backup exists, the retention policy is set to 1d and yet the WALs are never deleted. Am I missing something?

**Comment by kingfishx23** [comment]
Same. Can anyone enlighten us? Mine is set for 7d, but the WALs keep increasing. the base backups are also not deleted
